composer/datasets/ade20k_hparams.py

```diff
@@ -48,6 +49,9 @@ class ADE20kDatasetHparams(DatasetHparams, SyntheticHparamsMixin):
     final_size: int = hp.optional('Final size of the image and target', default=512)
     ignore_background: bool = hp.optional('If true, ignore the background class in training loss', default=True)
 
+    datadir: Optional[str] = hp.optional('The path to the data directory', default=None)
+    is_train: bool = hp.optional('Whether to load the training data (the default) or validation data.', default=True)
```

Comment:
I don't think this class needs `is_train`

Reply:
the constructor is invoked with `is_train=False` in `test_dataset_registry.py`

Reply:
Can we fix that in the tests? Would prefer not to keep non-test codes around to support tests.

Reply:
yes